src/PowerShellEditorServices/Language/FindDeclartionVisitor.cs

```diff
@@ -14,25 +14,31 @@ namespace Microsoft.PowerShell.EditorServices
     internal class FindDeclartionVisitor : AstVisitor
```

Comment:
Mind fixing the typo in the name of this class? Find*Declaration*Visitor.  That's bugged me for a while :)